client/blocks/eligibility-warnings/index.tsx

```diff
@@ -357,7 +363,7 @@ const processMarketplaceExceptions = (
 };
 
 const mapStateToProps = ( state: Record< string, unknown >, ownProps: ExternalProps ) => {
-	const siteId = getSelectedSiteId( state );
+	const siteId = getSelectedSiteId( state ) || ownProps.siteId || null;
```

Comment:
Very nitpicky but I would use `??` instead of `||` the same way as below.

Reply:
Do you mean something like `getSelectedSiteId( state ) ?? ownProps.siteId`? 

In that case, that would change the behavior. In this expression, I want to return either `getSelectedSiteId( state )` or `ownProps.siteId` or `null`. Using the `??` could return `undefined` if `ownProps.siteId` is `undefined`.

Please let me know if you meant something else.

Reply:
I was actually suggesting something like `getSelectedSiteId( state ) ?? ownProps.siteId ?? null`.
Completely non-blocking though.

Reply:
Ahms, ok, that will indeed keep the behavior. In this specific case, I see the `||` operator as a sum of values and multiple `??` operators like nesting conditions so for multiple values, it's clearer to me the `||` operator. Happy to change it if you feel strongly about the `??` though.

Reply:
The only difference is the OR operator `||` uses the right value if the left is falsy, while the nullish coalescing operator `??` uses the right value if left is null or undefined. 

I think the only actual difference is when we receive `0`, `''` or `false` that are falsy but not nullish.

I read it as `||` to check if its falsy and `??` to check if this is *empty*. So in my mind it makes sense but feel free to follow your gut feeling here.